.github/workflows/build_and_deploy.yml

```diff
@@ -504,21 +504,16 @@ jobs:
 
   releaseStats:
     name: Release Stats
-    runs-on: ubuntu-latest
+    runs-on:
+      - 'self-hosted'
+      - 'linux'
+      - 'x64'
+      - 'metal'
     needs: [publishRelease]
     steps:
-      - name: Setup node
-        uses: actions/setup-node@v3
-        with:
-          node-version: ${{ env.NODE_LTS_VERSION }}
```

Comment:
Are you sure we don't want to set a specific version of node?

Reply:
I think it might break if we use latest.

- See https://github.com/vercel/next.js/pull/54000

Reply:
That's controlled by the `next-stats-action` step instead of at the top level since we don't need to build until the action runs.

Reply:
https://github.com/vercel/next.js/blob/e471a46a2844ab4f13bfda7eceaf000ec2c59c24/.github/actions/next-stats-action/Dockerfile#L12-L13